Vagrantfile.example

```diff
@@ -35,6 +25,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     }
  end
 
+ # Comment this line if you would like to disable the automatic update during provisioning
+ config.vm.provision "shell", inline: "sudo dnf upgrade -y"
+
+ # bootstrap and run with ansible
+ config.vm.provision "shell", path: "playpen/vagrant-bootstrap-ansible.sh"
```

Comment:
Can we name this script "bootstrap-ansible" so it makes more sense when used by dev-setup.sh?